extensions/wikia/ArticleVideo/ArticleVideoController.class.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+class ArticleVideoController extends WikiaController {
+	public function index() {
+		$wg = F::app()->wg;
+
+		$articleId = RequestContext::getMain()->getTitle()->getArticleID();
+		$thumbnailPath = $wg->videoMVPArticles[$wg->cityId][$articleId]['thumbnailPath'];
+
+		$this->setVal( 'thumbnailUrl', $wg->extensionsPath . $thumbnailPath );
+
+		// TODO: replace it with DS icon when it's ready (XW-2824)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)